br/pkg/stream/models.go

```diff
@@ -44,7 +46,8 @@ func RangesOf(name string) string {
 // Normally it would be <prefix>/ranges/<task-name(string)>/<start-key(binary)> -> <end-key(binary)>
 func RangeKeyOf(name string, startKey []byte) string {
 	// We cannot use path.Join if the start key contains patterns like [0x2f, 0x2f](//).
-	return RangesOf(name) + string(startKey)
+	//return RangesOf(name) + string(startKey)
+	return path.Join(RangesOf(name), string(startKey))
```

Comment:
Please either use the origin version `RangesOf(name) + string(startKey)` or remove the comment `// We cannot use path.Join if the start key contains patterns like [0x2f, 0x2f](//).`